app/controller.go

```diff
@@ -268,8 +263,6 @@ func (app *App) txChecker() txChecker {
 			Codespace: "",
 		}
 
-		app.resetInternalState(app.Context.check)
```

Comment:
This could clear the whole journal for the blockEnder which may cause a hash id error.